src/redux/bookSlice.js

```diff
@@ -1,11 +1,35 @@
 import { createSlice } from '@reduxjs/toolkit';
 
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
+
 const bookSlice = createSlice({
   name: 'books',
-  initialState: [],
+  initialState,
   reducers: {
-    addBook: (state, action) => [...state, action.payload],
-    removeBook: (state, action) => state.filter((book) => book.id !== action.payload),
+    addBook: (state, action) => {
+      state.push(action.payload);
+    },
+    // removeBook: (state, action) => state.filter((book) => book.item_id !== action.payload),
```

Comment:
- Kindly, Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.  👍

Reply:
Removed. Thanks